task-management/src/main/webapp/vue-app/taskDrawer/components/TaskDrawer.vue

```diff
@@ -447,6 +447,7 @@ export default {
           this.task.status = value;
           this.$taskDrawerApi.updateTask(this.task.id, this.task).then( () => {
             this.oldTask.status = this.task.status;
+            this.$root.$emit('update-task-status',this.task);
```

Comment:
Could you use the same pattern as other event names. task-[propertyName]-updated